test/jasmine/tests/parcoords_test.js

```diff
@@ -0,0 +1,821 @@
+var Plotly = require('@lib/index');
+var Lib = require('@src/lib');
+var d3 = require('d3');
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+
+var mock0 = { // mock with zero dimensions; special case, as no dimension can be rendered
```

Comment:
I didn't try to run it outside the `docker` environment so I don't know. Are you suggesting it didn't run in `docker` because it hasn't yet got the `toSVG` method yet?